domain/src/main/java/com/github/maleksandrowicz93/educational/institution/CourseCloserModel.java

```diff
@@ -0,0 +1,43 @@
+package com.github.maleksandrowicz93.educational.institution;
+
+import com.github.maleksandrowicz93.educational.institution.api.infrastructure.CourseCloserAggregate;
+import com.github.maleksandrowicz93.educational.institution.enums.CourseState;
+import com.github.maleksandrowicz93.educational.institution.results.CourseClosingResult;
+import com.github.maleksandrowicz93.educational.institution.vo.CourseId;
+import com.github.maleksandrowicz93.educational.institution.vo.CourseSnapshot;
+import com.github.maleksandrowicz93.educational.institution.vo.Threshold;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+class CourseCloserModel extends CourseCloserAggregate {
```

Comment:
Nie rozumiem troszkę tego zamysłu, że masz zawsze abstract agregat i Model to rozszerza :) ?